index.js

```diff
@@ -2,29 +2,79 @@
  * Q0: Write a function called `countdown` that counts down from 10 on the console and
  * then logs "Blast Off!"
  */
+ function countdown() {
+ 	for (var i = 10; i >= 0; i--) {
+ 		console.log(i);
+ 		if (i <= 0) {
+	 		console.log('Blast off!!!');
+	 	}
+ 	}
+ }
+ countdown();
 
 
 /**
  * Q1: Write a function called `max` that has two Number parameters and returns
  * the larger of the two.
  */
+ function max(a, b) {
+ 	if (a > b) {
+ 		console.log(a);
+ 	} else if (a === b) {
+ 		console.log(a + ' & ' + b);
+ 	} else {
+ 		console.log(b);
+ 	}
+ }
+ max(5, 7);
+ max(4 * 7, (6 * 4) + 4);
+ max(4 * 7, 6 * 3);
+ max('five', 'ten');
 
 
 /**
  * Q2: Write a function called `longer` that has two String parameters and returns
  * the String that is longer.
  */
+ function longer(stringOne, stringTwo) {
+ 	var a = stringOne.length;
+ 	var b = stringTwo.length;
+ 	if (a > b) {
+ 		console.log(stringOne);
+ 	} else if (a === b) {
+ 		console.log("It's a draw!");
+ 	} else {
+ 		console.log(stringTwo);
+ 	}
+ }
```

Comment:
Same as above, this question is asking for you to `return` the value rather than logging it. Remember that returning a value from a function allows us to store that value in a variable or use it in another part of our program.